common/changes/xxsFacepile_2017-01-24-21-47.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Added new XXS Persona size, also usable in Facepile.. Made accessibility improvements to Persona and Facepile controls.",
```

Comment:
First time using the rush CLI to generate this.  I'll update it manually.  Thanks.